src/net/sourceforge/kolmafia/session/StillSuitManager.java

```diff
@@ -0,0 +1,57 @@
+package net.sourceforge.kolmafia.session;
+
+import java.util.ArrayList;
+import java.util.regex.Pattern;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.Modifiers;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.persistence.DebugDatabase;
+import net.sourceforge.kolmafia.preferences.Preferences;
+
+public class StillSuitManager {
+  private StillSuitManager() {}
+
+  public static void clearSweat() {
+    Preferences.setInteger("familiarSweat", 0);
+  }
+
+  public static void handleSweat(final String responseText) {
+    if (!responseText.contains("tiny_stillsuit.gif")) {
+      return;
+    }
+
+    var familiar =
+        KoLCharacter.familiars.stream()
+            .filter(f -> f.getItem() != null && f.getItem().getItemId() == ItemPool.STILLSUIT)
+            .findFirst()
+            .orElse(null);
+
+    int drams = 0;
+    if (familiar == KoLCharacter.getFamiliar()) {
+      drams = 2;
```

Comment:
```suggestion
      drams = 3;
```

Reply:
unless I am wrong about what this is, equipped familiars give 3 drams of sweat?

Reply:
You're right! I forgot